config/routes.rb

```diff
@@ -6,7 +6,14 @@
 	get "/login", to: "sessions#new"
 	post "/login", to: "sessions#create"
 
+	resources :users do
+	  member do
+	    get :following, :followers
+	  end
+	end
```

Comment:
@nozomi-matsuoka Remove this area because it is not used

Reply:
Thank you. I revised this file.